streams/src/test/java/org/apache/kafka/streams/integration/GlobalThreadShutDownOrderTest.java

```diff
@@ -146,7 +146,7 @@ public void shouldFinishGlobalStoreOperationOnShutDown() throws Exception {
             public boolean conditionMet() {
                 return firstRecordProcessed;
             }
-        }, 5000L, "Has not processed record within 5 seconds");
+        }, "Has not processed record within 15 seconds");
```

Comment:
When not time specified for the timeout, 15 seconds is used by default

Reply:
Nit: what if the default changes :P

Reply:
With regard to #5840 -- should we set timeout to 30 seconds?

Reply:
Yeah I'll update this to have the same timeout as well cf #5843